JSDemos/Demos/Form/Overview/description.md

```diff
@@ -1 +1,20 @@
-The Form component allows you to&nbsp;visually represent and edit data stored in&nbsp;an&nbsp;object. The component binds each layout item to&nbsp;a&nbsp;specific field in&nbsp;an&nbsp;existing data object. To&nbsp;display a&nbsp;simple form, you can assign the required object to&nbsp;the **formData** property. With the Form component, you can specify the location of&nbsp;labels against corresponding editors and number of&nbsp;columns used in&nbsp;the form layout. Optionally, the component can determine the column count based on&nbsp;current form width (adaptive rendering).
\ No newline at end of file
+The Form component represents fields of a data object as a collection of label-editor pairs. To create a data entry UI, assign an object to the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) property. The Form creates a simple item (a label-editor pair) for each field in the **formData** object.
+
+You can use the editors on the right to modify the following properties:     
+- [labelLocation](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelLocation)    
+Specifies whether to display item labels above the editor or to the left of it.
+
+- [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount)  
+Specifies the number of columns. Assign a number to this property to make the amount of columns fixed. Assign *"auto"* to make it adapt to the screen width.
```

Comment:
'amount' is used with uncountable nouns. 
First and second sentences are really similar - I think one is enough. 

```suggestion
Specifies the number of columns in the layout. To build an adaptive layout where column count depends on container width, set this property's value to *"auto"*.
```